src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -1034,7 +1035,9 @@ subroutine ocn_tend_freq_filtered_thickness(tendPool, statePool, meshPool, timeL
       call mpas_pool_get_array(meshPool, 'areaCell', areaCell)
       call mpas_pool_get_array(meshPool, 'edgesOnCell', edgesOnCell)
       call mpas_pool_get_array(meshPool, 'edgeSignOnCell', edgeSignOnCell)
+      call mpas_pool_get_array(meshPool, 'minLevelCell', minLevelCell)
       call mpas_pool_get_array(meshPool, 'maxLevelCell', maxLevelCell)
+      call mpas_pool_get_array(meshPool, 'minLevelEdgeBot', maxLevelEdgeBot)
```

Comment:
Is there possibly a `min` vs `maxLevelEdgeBot` mismatch here?

Also as a general question, I'm wondering how the `mpas_pool_get_array` type calls interact with the `use ocn_mesh` pattern? If a module is set up to `use ocn_mesh` does this mean we shouldn't have any `mpas_pool_get_array` calls for arrays from the mesh pool, with everything instead coming directly from the `ocn_mesh` accelerator?

Reply:
Yeah, that looks like a typo.

Matt Turner may be able to explain `ocn_mesh` better but I think the idea is to use it whenever possible, and not to use the pool calls.  But apparently there are some cases where it's not initialized yet or just not converted over yet?  Not sure in this particular case.

Reply:
Yes, thanks for catching it!
```suggestion
      call mpas_pool_get_array(meshPool, 'minLevelEdgeBot', minLevelEdgeBot)
```
Yes, we shouldn't need any of the `mpas_pool_get_array` calls when `use ocn_mesh` is employed after initialization. However, I'm not doing any of this clean-up with this PR as the norm is to keep the clean-up separate.